AcadeMeData/tests/test_client.py

```diff
@@ -1,13 +1,18 @@
 from AcadeMe import views
+from django.template.loader import render_to_string
 
 
 def test_homepage(client):
     request = client.get('')
     response = views.homePage(request)
+    expected_html = render_to_string('landing/homepage.html', request=request)
     assert response.status_code == 200
+    assert response.content.decode() == expected_html
 
 
 def test_contactus(client):
     request = client.get('contact_us/')
     response = views.contact(request)
+    expected_html = render_to_string('landing/contact_us.html', request=request)
     assert response.status_code == 200

```

Comment:
It should be split to two tests once for client request, the other for the view generate the HTML.

Reply:
Done